looper/run.sh

```
@@ -0,0 +1,177 @@
+# worktree の削除・マージは Verifier エージェントが担当。
+#
+# 使い方:
+#   bash looper/run.sh              # 実行
+#   bash looper/run.sh --dry-run    # 実行計画の確認のみ
+
+set -euo pipefail
+
+# スリープ防止（API接続断によるタイムアウトを回避）
+caffeinate -s -w $$ &
+CAFFEINATE_PID=$!
+trap "kill $CAFFEINATE_PID 2>/dev/null" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)